newIDE/app/src/Utils/GDevelopServices/Usage.js

```diff
@@ -467,6 +396,28 @@ export const getRedirectToCheckoutUrl = ({
   return url.toString();
 };
 
+export const getRedirectToCheckoutUrlV2 = ({
```

Comment:
is the v1 still used?

Reply:
Yes, it will be used by all the previous versions. We will remove it at some point.

Reply:
I'm not sure I understand, which previous versions? GDevelop? If that's the case we don't need it as when you merge it's not used anymore

Reply:
Oh yeah you're right, I'll remove it